spdx-utils/build.gradle.kts

```diff
@@ -358,7 +358,10 @@ val generateLicenseRefTextResources by tasks.registering {
         licensesDir.walk().maxDepth(1).filter {
             it.isFile && it.extension == "yml" && !it.nameWithoutExtension.endsWith("-exception")
         }.forEach { file ->
-            val isSpdxLicense = file.readLines().any { it.startsWith("spdx_license_key: ") }
+            val isSpdxLicense = file.readLines().any {
+                it.startsWith("spdx_license_key: ") && !it.contains("LicenseRef-")
```

Comment:
BTW, the single space (" ") I wrote before "LicenseRef-" was there intentionally, but I believe this is also good enough.

Reply:
I just haven't noticed that one.